cookbooks/tomography_based_plate_motions/doc/tomography_based_plate_motions.md

```diff
@@ -65,3 +65,47 @@ The user can change several parameters to investigate the mantle forces and how
 - The reference viscosity profile.
 - The prescribed viscosity of the plate boundaries, which controls friction between plates.
 - To use viscous and neutrally buoyant cratons in the model.
+
+# Added complexity—Slab2 and initial topography
+For additional complexity, we use the Slab2 {cite}`hayes2018slab2` database that describes the three dimensional geometries of all the seismically active subduction zones in detail on Earth instead of the vertical slabs defined in the TM1 model. We also deform the outer shell by the initial topography. Incorporating these finer heterogeneities allow us to simulate Earth-like surface deformation in the models.
+
+The slabs are included as a compositional field that is diffused with a length scale of 30 km. Physically, this value represents the diffusion of a slab in $\approx$ 15 Ma using diffusivity of 1.14e{-6} m$^2$/s$^{-1}$. Within the slabs, we compute the temperatures using the diffused slabs' composition and a user-defined temperature anomaly, added to the reference mantle adiabat. Additionally, we add a low-viscosity layer in the material model, the mid-mantle layer, that extends below the mantle transition zone until 1000 km depth. The weak mid-mantle layer ensures that our slabs do not get stuck and can still pull the attached lithospheric plates as they sink into the more viscous lower mantle.
```

Comment:
Can you add a description for why the slabs are diffused in this model (improve solver stability, more physically realistic, etc)?